x-pack/docs/en/security/troubleshooting.asciidoc

```diff
@@ -319,6 +320,65 @@ In this case, you must install the
 <<ciphers, JCE Unlimited Strength Jurisdiction Policy Files>>.
 --
 
+[[trb-security-kerberos]]
+=== Common Kerberos exceptions
+
+*Symptoms:*
+
+* User authentication fails due to either GSS negotiation failure
+or a service login failure on the server side or in the {es} HTTP client. Some of
+the common exceptions are listed below with some tips to resolve them. 
+
+*Resolution:*
+
+`Failure unspecified at GSS-API level (Mechanism level: Checksum failed)`::
++
+--
+
+When this occurs on HTTP client side, it may be related to an incorrect password.
+
+The keytab is present but it failed to log in as user. Check the keytab expiry. 
+Also check whether the keytab contain up-to-date credentials; if not, replace them.
+
+Use tools like `klist` or `ktab` to list the principals inside the keytab and
+validate them.
+
+Kerberos depends on proper hostname resolution, so check your DNS setup for 
+missing configurations like incorrect DNS SRV records or incorrect configuration 
+for the Key Distribution Center(KDC) servers in `krb5.conf`.
+
+--
+
+`Failure unspecified at GSS-API level (Mechanism level: Request is a replay (34))`::
+
+`Failure unspecified at GSS-API level (Mechanism level: Clock skew too great (37))`::
++
+--
+
+To prevent replay attacks, Kerberos V5 sets maximum tolerance for computer clock
+synchronization and it is usually set to 5 minutes. Check whether the time on 
+the machines within the domain is in sync.
+
+--
+
+For further debugging, enable JAAS Kerberos login module debugging logs on
+{es} by using following settings:
+
+[source,yaml]
+----------------
+xpack.security.authc.realms.<realm-name>.krb.debug: true
+----------------
+
+For detailed information, see {ref}/security-settings.html#ref-kerberos-settings[Kerberos realm settings].
+
+To enable Kerberos logging on JVM, add following JVM system properties:
```

Comment:
I think this needs some context. Like when and why would you do this. How does it differ from the login module debug log